website/integrations/services/mastodon/index.md

```diff
@@ -48,7 +48,7 @@ OIDC_DISCOVERY=true
 OIDC_ISSUER=< OpenID Configuration Issuer>
 OIDC_AUTH_ENDPOINT=https://authentik.company/application/o/authorize/
 OIDC_SCOPE=openid,profile,email
-OIDC_UID_FIELD=sub
+OIDC_UID_FIELD=preferred_username
```

Comment:
I thin it would be nice to explain bpth options to the readers. Maybe make `preferred_username` be the one in this list, but then below the list you could add a sentence to explain that one can use `OIDC_UID_FIELD=sub` if you want names generated?